### PR TITLE
Update environment-variables

### PR DESCRIPTION
**Pull request by ian28223**
### What does this PR do?
Fix some 404 and document https://github.com/DataDog/datadog-agent/pull/16017

### Motivation
- document new feature

### Additional Notes
- No translated version of this change for for ES, FR, and JA

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/agent/guide/environment-variables.md [hunk 1]
```diff
@@ -91,14 +91,30 @@ In general, use the following rules:
              # DD_PROCESS_AGENT_URL=https://process.datadoghq.com
           ```
 
+## Using environment variables in systemd units
+
+In Operating Systems that uses SystemD to manage services, environment variables - global (e.g. `/etc/environment`) or session-based (e.g. `export VAR=value`) - are not generally made available to services unless configured to do so. See [SystemD Exec man page][8] for more details.
```
Comment (brett0000FF) [inline content/en/agent/guide/environment-variables.md:96]:
```suggestion
In operating systems that uses systemd to manage services, environment variables—global (for example, `/etc/environment`) or session-based (for example, `export VAR=value`)—are not generally made available to services unless configured to do so. See [systemd Exec manual page][8] for more details.
```

File: content/en/agent/guide/environment-variables.md [hunk 2]
```diff
@@ -91,14 +91,30 @@ In general, use the following rules:
...
+
+From Datadog Agent 7.45,the Datadog Agent service (`datadog-agent.service` unit) can optionally load environment variables assignments from a file (`<ETC_DIR>/environment`).
```
Comment (brett0000FF) [inline content/en/agent/guide/environment-variables.md:98]:
```suggestion
From Datadog Agent 7.45, the Datadog Agent service (`datadog-agent.service` unit) can optionally load environment variables assignments from a file (`<ETC_DIR>/environment`).
```

File: content/en/agent/guide/environment-variables.md [hunk 3]
```diff
@@ -91,14 +91,30 @@ In general, use the following rules:
...
+
+- Create `/etc/datadog-agent/environment` if it does not exist.
```
Comment (brett0000FF) [inline content/en/agent/guide/environment-variables.md:100]:
```suggestion
1. Create `/etc/datadog-agent/environment` if it does not exist.
```

File: content/en/agent/guide/environment-variables.md [hunk 4]
```diff
@@ -91,14 +91,30 @@ In general, use the following rules:
...
+- Define newline-separated environment variable assignments. Example:
```
Comment (brett0000FF) [inline content/en/agent/guide/environment-variables.md:101]:
```suggestion
2. Define newline-separated environment variable assignments. Example:
```

File: content/en/agent/guide/environment-variables.md [hunk 5]
```diff
@@ -91,14 +91,30 @@ In general, use the following rules:
...
+  ```
+  GODEBUG=x509ignoreCN=0,x509sha1=1
+  DD_HOSTNAME=myhost.local
+  DD_TAGS=env:dev service:foo
+  ```
+- Restart the service for changes to take effect
```
Comment (brett0000FF) [inline content/en/agent/guide/environment-variables.md:107]:
```suggestion
3. Restart the service for changes to take effect
```

**Review by brett0000FF — CHANGES_REQUESTED** [review]
Thanks for this content! I left a couple small suggestions.

**Review by brett0000FF — APPROVED** [review]
Looks good! Let me know if you need me to merge for you.

**Comment by ian28223** [comment]
> Looks good! Let me know if you need me to merge for you.

Yes, please; I'm not authorised to merge. 
 Much appreciated!

**Pull request merged**